.github/workflows/ci_workflows.yml

```diff
@@ -66,15 +66,15 @@ jobs:
           linux: py311-test-cov
           coverage: codecov
 
-        - name: Python 3.10 with all optional dependencies
+        - name: Python 3.10 in parallel with all optional dependencies
           linux: py310-test-alldeps-fitsio
           libraries:
             apt:
               - language-pack-fr
               - tzdata
               - libcfitsio-dev
           toxargs: -v --develop
-          posargs: --run-slow
+          posargs: -n=4 --run-slow
```

Comment:
@astrofrog, I moved the parallel test to the all dependencies test. Do you like this better?

Reply:
@astrofrog , please re-review and confirm. Thanks!